vowpalwabbit/fb_parser/tests/flatbuffer_parser_test.cc

```diff
@@ -28,40 +28,85 @@ flatbuffers::Offset<VW::parsers::flatbuffer::ExampleRoot> sample_flatbuffer_coll
 {
```

Comment:
Is this method called from a test?

Reply:
No as of now. Will update the code to include it